edx_solutions_api_integration/users/test_users.py

```diff
@@ -889,7 +889,7 @@ def test_user_detail_invalid_email(self):
         }
         response = self.do_post(test_uri, data)
         self.assertEqual(response.status_code, 400)
-        self.assertIn('Invalid email address', response.content)
+        self.assertIn(b'Invalid email address', response.content)
```

Comment:
@wasifarbisoft can we use `.decode("utf-8")` here instead of converting the string to bytes?

Reply:
Fixed!